### PR TITLE
Implement seccomp profile generator #2

### PR DESCRIPTION
**Pull request by TakayukiBGoto**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -31,6 +31,11 @@ fn main() {
         .subcommand(
             SubCommand::with_name("report")
                 .about("Dump analyzer result")
+                .arg(
```
Comment (KentaTada) [inline src/main.rs]: I think that seccomp-profile also should take the value to specify the generated json file name with --output. default_value of seccomp-profile should be set like "seccomp.json". If users set --output and --seccomp-profile at the same time, report subcommand should generate both raw output file and seccomp.json.

**Comment by KentaTada** [comment]
Thank you @TakayukiBGoto 

- I think that report subcommand should generate both raw output file and seccomp.json when users set --output and --seccomp-profile at the same time.
- I think that --dump option should not be used with --seccomp-profile. In addition to that, I'm sorry but --dump was not obvious.  Could you change the name from --dump to --livedump?
- BTW, we need to get rid of unused syscalls which are executed during runC startup.  Is it enough to get rid of syscalls before exec()? I think we will continue to consider this issue in future.

**Comment by TakayukiBGoto** [comment]
@KentaTada Thank you for your comment.
I modified all your pointed out matters.
Please review again.

**Comment by KentaTada** [comment]
I'm sorry but I misunderstood. Current implementations uses the events of syscalls but these events could not trace correct syscalls.  We should use raw_syscalls although we rejected that because it was difficult to convert system call numbers into names.

We will use raw_syscalls and use ausyscall to convert system call numbers.

**Comment by KentaTada** [comment]
This request was merged at https://github.com/KentaTada/oci-ftrace-syscall-analyzer/pull/13

**Pull request closed without merging**